charts/shoot-core/charts/vpn-shoot/templates/deployment.yaml

```diff
@@ -35,6 +35,19 @@ spec:
       tolerations:
       - key: CriticalAddonsOnly
         operator: Exists
+{{- if .Values.initContainers }}
+      initContainers:
+      {{- range $index, $ctr := .Values.initContainers }}
+      - name: init-{{ $index }}
+        image: {{ $ctr.image }}
+        {{- if $ctr | .command }}
+        command: {{ $ctr.command }}
```

Comment:
```suggestion
        command: {{ $ctr.command }}
        command:
{{ toYaml $ctr.command | indent 8 }}
```

Reply:
Would this work if the value of `$ctr.command` is structured as a `["a","b"]` array? I never was sure about it. Will accept as you recommended.

Reply:
Yes. Try it out: `helm install charts/shoot-core/charts/vpn-shoot --debug --dry-run --name test`

Reply:
Yup. Thanks.